clang/include/clang/Serialization/ASTBitCodes.h

```diff
@@ -1079,6 +1079,15 @@ enum PredefinedTypeIDs {
   /// \brief The '__ibm128' type
   PREDEF_TYPE_IBM128_ID = 74,
 
+  /// \brief The '_Decimal32' type
+  PREDEF_TYPE_DECIMAL32_ID = 75,
+
+  /// \brief The '_Decimal64' type
+  PREDEF_TYPE_DECIMAL64_ID = 76,
+
+  /// \brief The '_Decimal128' type
+  PREDEF_TYPE_DECIMAL128_ID = 77,
+
```

Comment:
We'll need to determine if this change requires a change to `VERSION_MAJOR` or `VERSION_MINOR` above (around line 50)

Reply:
How are we going to know that?

Reply:
Good question :) The best idea I have is to craft a source program that generates an AST that exercises the OpenCL image types below (that presumably use the type IDs stolen above), generate a .ast file using a build of Clang that doesn't have these changes, then consume the .ast file with a build that has these changes, and see what happens. If the .ast file is rejected with an appropriate diagnostic indicating an incompatibility, then great. If the .ast file is accepted, but a dump of the AST now shows the wrong types or otherwise behaves badly, then it seems a version bump is needed.

Reply:
I filed issue #17 to follow up on this.